### PR TITLE
0624 acb asyncapi duke (with base latest)

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 1]
```diff
@@ -263,10 +263,20 @@ When the application deploys successfully, you see a `DEPLOYED` log message in t
...
+
+If you add an OAS fragment or JSON schema fragment from Exchange to a project and then use it in an API spec, Anypoint Code Builder fails to scaffold the API.
+--
 // end::fragments-not-scaffolded[]
 //
-// If you add an OAS fragment or JSON schema fragment from Exchange to a project and then use it in an API spec, Anypoint Code Builder fails to scaffold the API.
+
+//
+// tag::open-beta-asyncapi[]
+// AsyncAPI scaffolding feature in open Beta from Feb 2024 until the Oct 2024
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc]: TBD if this include is the tack to take.

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 2]
```diff
@@ -1,18 +1,20 @@
-= Implementing OAS, RAML, and GraphQL APIs
+//TODO: Propose something like this:
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc:1]: TBD: Consider a shorter title as suggested in comments.

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
 into a Mule project for your implementation or integration.
 
 [IMPORTANT]
 include::partial$acb-reusable-steps.adoc[tags="fragments-not-scaffolded"]
 
-Anypoint Code Builder uses the information that you defined in your spec to 
+Anypoint Code Builder uses the configuration defined in the spec to 
 autogenerate a basic interface in your implementation project.
-This interface includes flows for each endpoint, a built-in XML-based router, and error handlers,
+This interface includes flows for each endpoint or operation, a built-in XML-based router, and error handlers,
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc:19]: TODO: check that it's the operations or what term to use here (flow per operation?)

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 4]
```diff
@@ -1,18 +1,20 @@
-= Implementing OAS, RAML, and GraphQL APIs
+//TODO: Propose something like this:
+//= Implementing Synchronous and Asynchronous APIs
+= Implementing OAS, RAML, AsyncAPI, and GraphQL APIs
 :page-deployment-options: cloud-ide, desktop-ide
 
 include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
-
+include::anypoint-code-builder::partial$acb-reusable-steps.adoc[tag="open-beta-asyncapi"]
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc]: This one is overkill and covered by a simple addition of AsyncAPI to the existing notification. 

```suggestion
```

File: modules/ROOT/pages/imp-implement-local-apis.adoc [hunk 5]
```diff
@@ -2,7 +2,7 @@
 :page-deployment-options: cloud-ide, desktop-ide
 
 include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"] 
-
+include::anypoint-code-builder::partial$acb-reusable-steps.adoc[tag="open-beta-asyncapi"]
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-local-apis.adoc]: same comment as above
```suggestion
```

File: modules/ROOT/pages/imp-implementing-apis.adoc [hunk 6]
```diff
@@ -4,14 +4,12 @@
 // implement-api-specification
 
 include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"] 
-
-//LOGO (web, desktop, or both)
-// include::partial$acb-ide-logos.adoc[tags="both-ides"]
+include::anypoint-code-builder::partial$acb-reusable-steps.adoc[tag="open-beta-asyncapi"]
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implementing-apis.adoc]: same comment as above
```suggestion
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 7]
```diff
@@ -1,18 +1,20 @@
-= Implementing OAS, RAML, and GraphQL APIs
+//TODO: Propose something like this:
+//= Implementing Synchronous and Asynchronous APIs
+= Implementing OAS, RAML, AsyncAPI, and GraphQL APIs
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc]: TBDs
```suggestion
= Implementing APIs
```

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 8]
```diff
@@ -263,10 +263,23 @@ When the application deploys successfully, you see a `DEPLOYED` log message in t
...
-// If you add an OAS fragment or JSON schema fragment from Exchange to a project and then use it in an API spec, Anypoint Code Builder fails to scaffold the API.
+// tag::fragments-inline[]
+
+For OAS and JSON schema, you can also specify fragment files that you create manually within the implementation or integration project directory.
+// end::fragments-inline[]
+//
+
+
+//
+// NOT USED
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc:277]: Don't review open-beta-asyncapi -- NOT USED

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 9]
```diff
@@ -446,3 +459,70 @@ Although it is also possible to set the versions as values to *muleRuntimeVersio
...
+.. Clicking *Get Access*
++ 
+When the AsyncAPI feature is available for your organization, the Anypoint Code Builder landing page provides the following notification:
++
+*AsyncAPI Implementation Beta for Anypoint Code Builder and Studio is enabled for your organization. Tell your team to learn more.*
+// end::asyncapi-beta-access[]
+//
+
+//
+// tag::prereqs-implementation[]
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc:484]: DON'T REVIEW tag::prereqs-implementation -- converts already reviewed content into an include tag

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 10]
```diff
@@ -446,3 +459,70 @@ Although it is also possible to set the versions as values to *muleRuntimeVersio
...
+* xref:start-acb.adoc[Set up and access the desktop or cloud IDE].
+* xref:des-create-api-specs.adoc[Create and test your API spec].
+* xref:des-publish-api-spec-to-exchange.adoc[Publish your API spec to Exchange].
+* Optionally, xref:int-create-integrations.adoc[create your integration project (Mule app)].
+// end::prereqs-implementation[]
+// 
+
+//
+// tag::start-implementation[]
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc:493]: DON'T REVIEW tag::start-implementation -- converts already reviewed content into an include tag

File: modules/ROOT/pages/imp-asyncapi.adoc [hunk 11]
```diff
@@ -0,0 +1,790 @@
+= Implementing AsyncAPI Specifications
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-asyncapi.adoc:1]: Please REVIEW ALL (except code examples)

File: modules/ROOT/pages/imp-asyncapi.adoc [hunk 12]
```diff
@@ -0,0 +1,790 @@
...
+== Examples
+
+The <<scaffold, scaffolding procedure>> builds an implementation project from the following version 2.6 AsyncAPI spec:
+
+* <<ex-asyncapi-spec>>
++
+The AsyncAPI specification TODO_TODO
+* <<ex-flows-scaffolded>>
++
+TODO_TODO
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-asyncapi.adoc]:
```suggestion
```

File: modules/ROOT/pages/imp-asyncapi.adoc [hunk 13]
```diff
@@ -0,0 +1,790 @@
...
+* <<ex-connection-configs-scaffolded>>
++
+TODO_TODO
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-asyncapi.adoc]: TBD
```suggestion
```

File: modules/ROOT/pages/imp-asyncapi.adoc [hunk 14]
```diff
@@ -0,0 +1,790 @@
...
+* <<ex-config-properties>>
++
+TODO_TODO
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-asyncapi.adoc]:
```suggestion
```

File: modules/ROOT/pages/imp-implement-api-specs.adoc [hunk 15]
```diff
@@ -1,18 +1,22 @@
-= Implementing OAS, RAML, and GraphQL APIs
+//TODO: Propose something like this:
+//= Implementing Synchronous and Asynchronous APIs
+= Implementing OAS, RAML, AsyncAPI, and GraphQL APIs
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-api-specs.adoc]:
```suggestion
= Implementing OAS, RAML, and GraphQL APIs
```

**Review by shranga — APPROVED** [review]

**Comment by dukesphere** [comment]
This was merged into https://github.com/mulesoft/docs-code-builder/pull/327 to address some unverified commits.

**Pull request closed without merging**